index.js

```diff
@@ -1,20 +1,28 @@
 import process from 'node:process';
 import execa from 'execa';
 
-export async function runAppleScript(script) {
+const DEFAULT_OPTIONS = {
+	humanReadableOutput: true,
+};
+
+export async function runAppleScript(script, options = DEFAULT_OPTIONS) {
 	if (process.platform !== 'darwin') {
 		throw new Error('macOS only');
 	}
 
-	const {stdout} = await execa('osascript', ['-e', script]);
+	const outputArg = options.humanReadableOutput ? [] : ['-ss'];
```

Comment:
```suggestion
	const outputArguments = options.humanReadableOutput ? [] : ['-ss'];
```

Reply:
I feel like outputArgs is expressive enough but updated it in [da05ff8](https://github.com/sindresorhus/run-applescript/pull/14/commits/da05ff82061133e87977cfc2af0782a18c46356f)